modes/en/pdf.md

```
@@ -0,0 +1,151 @@
+a. `start-editing-transaction` on the duplicate design
+b. `perform-editing-operations` with `find_and_replace_text` for each section
+c. **Reflow layout after text replacement** to maintain even spacing between sections
+d. **Verify layout before commit** with `get-design-thumbnail`
+e. Show the user the final preview and ask for approval
+f. `commit-editing-transaction` to save (ONLY after user approval)
+
+#### Step 5 — Export and download PDF
+
+a. `export-design` the duplicate as PDF
+b. **IMMEDIATELY** download the PDF using Bash:
+   ```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)